prow/cluster/bazel-cache-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bazel-cache
+  namespace: ${NAMESPACE}
+spec:
+  selector:
+    matchLabels:
+      app: bazel-cache
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: bazel-cache
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: prow-master-03 # This could be a proper label such as `role: bazel-cache`
+
+      initContainers:
+      - name: setup-permissions
+        image: nginx:1.20 # Same image as the main container, keep them in sync
+        volumeMounts:
+          - name: cache
+            mountPath: /var/cache/bazel/cache
```

Comment:
why mount under `.../cache`? aren't we already prefixing with `/cache/` in the nginx config?

Reply:
ah I see, thanks!